.travis.yml

```diff
@@ -2,8 +2,8 @@ language: ruby
 rvm:
   - 2.3.0
 
-services:
-  - postgresql
+addons:
+  postgresql: "9.4"
```

Comment:
i changed the version because we're using a json field in our model, we will have to change it to string if we want to be on an old version